### ISSUE TITLE
Poor performance on the test set

### ISSUE THREAD

**Issue by TobiasLee** [jcaip/twitter_sentiment_analysis#1]
Thanks for sharing the code! But I met serious underfitting problem when using your code.
I ran the code on the whole dataset( 1,600,000 training data and 498 test data) ,however the test 
 error is 0.506964, and the prediction is all zeros(negative), far away from my expectation.
All I have done is modified the train.py a little bit - modifying the config as below to fit my dataset's location and size.

config = {
        'TRAINING_DATA_LOCATION': './trainingandtestdata/train.csv',
        'TESTING_DATA_LOCATION': './trainingandtestdata/test.csv',
        'NUM_EXAMPLES': 1600000,
        'BATCH_SIZE': 1000,
        'NUM_EPOCHS': 1,
        'LEARNING_RATE': 0.001,
        'HIDDEN_LAYER_SIZE': 150,
        'WORD_VECTOR_DIM': 200,
        'SAVE_LOCATION': os.path.join('./checkpoints/', str(time.time))
}
Is there is something wrong?  I'd appreciated it if you can help me, thanks!

The result : 
1600/1600 [3:40:56<00:00,  8.34s/it] 
Training Error: 0.468
Test Error: 0.506964
Predictions: [0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]

**Comment by TobiasLee** [comment]
Sorry to bother,  I made a stupid mistake.
In the loss function, I wrote `tf.sigmoid_cross_entropy_with_logits` instead of `tf.softmax_cross_entropy_with_logits`, causing by autocomplete.
